### DISCUSSION TITLE
rocm 6.3.0 doesnt work on rx 7600s?

### DISCUSSION THREAD

**Discussion by patricknassef1** [ROCm/ROCm#4188] [Q&A] [answered]
if any one know which version works with rx7600s bec i tried just tried rocm 6.3.0 and it didnt work

**Comment by darren-amd** [comment, marked as answer]
Hi @patricknassef1,

We currently do not support the RX 7600S officially: [Supported GPUs](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html). 

However, since it is based on the same RDNA 3 architecture as other supported cards, some users have gotten to work. Is there a particular workload that doesn't work for you? You could try overriding `export HSA_OVERRIDE_GFX_VERSION=11.0.0`, which would make ROCm report your GPU as gfx1100.

**Reply by patricknassef1** [reply]
thank you for your reply ! sadly i did override those 
export TORCH_USE_HIP_DSA=1
export HSA_OVERRIDE_GFX_VERSION=11.0.0
export HIP_VISIBLE_DEVICES=0
export AMD_SERIALIZE_KERNEL=3

and tried training a yolov8 model with but it kept saying :  Segmentation fault (core dumped)

i did a little research and found nothing about it.. and i made sure that the rocm reads my rx 7600s by using rocminfo and made sure that torch has CUDA enabled .. it's just the training part that keep failing..

**Reply by darren-amd** [reply]
Hi @patricknassef1,

Are you using Pytorch? If so, could you try following the [PyTorch Installation on ROCm](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.3.1/install/3rd-party/pytorch-install.html#using-docker-with-pytorch-pre-installed) using the docker image?

**Reply by patricknassef1** [reply]
i followed every single step of the installation and i did all of that and as i told you i made sure that torch CUDA detects my GPU and it did

**Reply by darren-amd** [reply]
Could you try testing your Pytorch installation as well as run a few of the Pytorch basic examples from [here](https://rocm.docs.amd.com/projects/install-on-linux/en/develop/install/3rd-party/pytorch-install.html#testing-the-pytorch-installation) to see if there is an issue with your installation?

**Comment by harkgill-amd** [comment]
We've made large strides in both GPU support since this discussion was first opened. Please give the latest release below a try and feel free to open a GitHub issue if you run into any difficulties.

https://rocm.docs.amd.com/en/latest/install/rocm.html
